.gitignore

```diff
@@ -1 +1,2 @@
 /target
+/deps
```

Comment:
Is this needed?

Reply:
there's always a `deps` folder that pops up, not sure from which tool : C++ maybe ?

Reply:
otherwise let me just remove it ...